### PR TITLE
PIM-7473 - Enable cache for validation mapping

### PR DESCRIPTION
**Pull request by BitOne**
**Description (for Contributor and Core Developer)**

A lot of screens are using validation (grid, pef, etc...), but every time a HTTP request for those screens is made, the validation configuration is read from disk and the YML is interpreted, leading to slowness.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | WIP
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/config/pim.yml [hunk 1]
```diff
@@ -57,3 +57,7 @@ pim_catalog:
             - { value: 'MM/dd/yyyy', label: 'mm/dd/yyyy' }
             - { value: 'dd/MM/yyyy', label: 'dd/mm/yyyy' }
             - { value: 'dd.MM.yyyy', label: 'dd.mm.yyyy' }
+
+framework:
+    validation:
+            cache: apc
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/config/pim.yml]: double indentation?

**Comment by jjanvier** [comment]
:ah:

Didn't know about that :(

**Comment by jjanvier** [comment]
Is it a noticeable boost?

**Comment by BitOne** [comment]
@jjanvier : The boost is between 50 and 100 ms per page on my machine. Not crazy, but still a little bit more than 10% on the grid.

**Pull request merged**